meson_options.txt

```diff
@@ -19,6 +19,7 @@ option('plugin_uefi', type : 'boolean', value : true, description : 'enable UEFI
 option('plugin_nvme', type : 'boolean', value : true, description : 'enable NVMe support')
 option('plugin_modem_manager', type : 'boolean', value : false, description : 'enable ModemManager support')
 option('plugin_flashrom', type : 'boolean', value : false, description : 'enable libflashrom support')
+option('plugin_coreboot', type : 'boolean', value : true, description : 'enable coreboot support')
```

Comment:
Coreboot can't default to true if flashrom defaults to false and coreboot needs it.